src/hotspot/share/jeandle/jeandleAbstractInterpreter.cpp

```diff
@@ -1420,3 +1420,27 @@ void JeandleAbstractInterpreter::arraylength() {
     llvm::CallInst* call = call_java_op("jeandle.arraylength", {array_oop});
     _jvm->ipush(call);
 }
+
+void JeandleAbstractInterpreter::newarray(int dtype){
+    llvm::Value* length = _jvm->ipop();
```

Comment:
These callee are slow path of allocation. We need implement fast path allocation in future. I think we can wrapper it as a java_op, and we can change the implementation in future.